app/controllers/admin/articles_controller.rb

```diff
@@ -0,0 +1,50 @@
+class Admin::ArticlesController < Admin::ApplicationController
+  def index
+    @articles = Article.all 
+  end
+
+  def create
+    @article = Article.new(article_params)
+    
+    if @article.save
+      redirect_to admin_articles_path, :status=> :found, :notice => 'You have successfully created a new article!'
+      
+    else
+      render 'new'
+    end
+  end
+  
+  def show
+    @article = Article.find(params[:id])
```

Comment:
We can clean this up further, but will be easier to do it with you, when you're next in the office :)